src/components/SwitchTheme/SwitchTheme.css

```diff
@@ -0,0 +1,15 @@
+.btn-theme-toggle {
+  background-color: #1a1a1a;
+  padding: 15px;
+  border-radius: 50%;
+  border: initial;
+  transition: initial;
+  font-family: initial;
+  font-size: 0em;
+  margin: 10px;
```

Comment:
seems like we have this css some where. after fix it you can make global "interface button" css class and use it everywhere